### ISSUE TITLE
Navigate twice when hitting space bar after clicked next button on OSC

### ISSUE THREAD

**Issue by yhatt** [marp-team/marp-cli#156]
A focus ring will be kept on the clicked next button on OSC, so space bar brings unexpected twice navigation.

It is same for the other buttons. For example, slide cannot navigate with space bar after clicked previous button. Buttons for bespoke OSC plugin should not keep focus ring.
